src/main/java/codeu/model/data/User.java

```diff
@@ -159,7 +240,7 @@ void createConversations() {
             }
 
             /* Creates conversation link by concatenating the two user's names in alphabetical order. */
-            String conversationName = firstUser + secondUser;
+            String conversationName = firstUser + "&" + secondUser;
```

Comment:
If you're going to use this in the url, better test to be sure it won't cause a problem (ie it gets urlencoded..) since the ampersand has special use in a url ..

Reply:
I see what you mean, I'll change it to "-" since only letters+numbers are allowed in conversation names that users can use, but the "-" seems to be a safe character in terms of using in URL path.